internal/cmd/helm-operator/run/cmd.go

```diff
@@ -128,6 +128,14 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 	// Set default manager options
 	options = f.ToManagerOptions(options)
 
+	// Log manager option flags
+	log.Info("Setting manager options:",
+		"MetricsBindAddress", options.MetricsBindAddress,
+		"HealthProbeAddress", options.HealthProbeBindAddress,
+		"LeaderElection", options.LeaderElection,
+		"LeaderElectionId", options.LeaderElectionID,
+		"LeaderElectionNamespace", options.LeaderElectionNamespace)
```

Comment:
Will it be confusing for the logs to include output for unset or unused flags?

For example, this is going to output something like: `LeaderElectionId="", LeaderElectionNamespace=""` even if leader election is disabled. Is that what we want?

Reply:
That's a good point. I think if leader election is disabled it is likely not important to log those values. I will work on making it so those values are only logged if they are set.

Reply:
As of commit https://github.com/operator-framework/helm-operator-plugins/pull/165/commits/37c3d87ec84068791a3d676d69ae80ef8a63a97c it should only show the options that have been specified via the flags or have a default value.

The output will now look like this with both leader election related flags:
```
{"level":"info","ts":1652113560.2827275,"logger":"cmd","msg":"Setting manager options:","Options":{"HealthProbeAddress":":8081","LeaderElection":true,"LeaderElectionId":"1","LeaderElectionNamespace":"test","MetricsBindAddress":":8080"}}
```

With only one:
```
{"level":"info","ts":1652113924.1076055,"logger":"cmd","msg":"Setting manager options:","Options":{"HealthProbeAddress":":8081","LeaderElection":true,"LeaderElectionId":"1","MetricsBindAddress":":8080"}}
``` 

With neither:
```
{"level":"info","ts":1652113954.4604402,"logger":"cmd","msg":"Setting manager options:","Options":{"HealthProbeAddress":":8081","LeaderElection":false,"MetricsBindAddress":":8080"}}
```